online_party_system/party_match.go

```diff
@@ -114,14 +114,18 @@ type PartyMatch struct {
 func (p *PartyMatch) cleanupExpiredInvitations(ctx context.Context, nk runtime.NakamaModule, s *PartyMatchState) error {
 	if p.config.InviteDuration.Nanoseconds() > 0 {
 		now := time.Now()
+		subject := "Party expired invitation"
 		for memberId, expiration := range s.invitations {
 			if now.After(expiration) {
 				delete(s.invitations, memberId)
 				content := map[string]interface{}{
 					"party_id":  ctx.Value(runtime.RUNTIME_CTX_MATCH_ID).(string),
 					"member_id": memberId,
 				}
-				if err := p.sendPartyNotification(ctx, nk, s, "Party expired invitation", content); err != nil {
+				if err := p.sendPartyNotification(ctx, nk, s, subject, content); err != nil {
+					return err
+				}
+				if err := nk.NotificationSend(ctx, memberId, subject, content, 1, "", false); err != nil {
```

Comment:
Can you move the notification code to a public const field with a number somewhat high (like in the 1000 range)? This will prevent collision where that notif code might be used by other consumers of the library already.

Reply:
that code is hardcoded below in sendPartyNotification() to 1, if I move it out from there and change it it will definitely break other consumers right?

Reply:
Nice catch. I think we're stuck with the notif codes we have for now. 👍